featureflags_test.go

```diff
@@ -0,0 +1,26 @@
+package posthog
+
+import (
+	"math"
+	"testing"
+)
+
+func TestCalculateHash(t *testing.T) {
+	for _, tt := range []struct {
+		ident string
+		want  float64
+	}{
+		{"some_distinct_id", 0.086639},
+		{"test-identifier", 0.749634},
+		{"example_id", 0.869139},
+		{"example_id2", 0.844273},
+	} {
```

Comment:
These look like the same cases we use in https://github.com/PostHog/posthog/blob/master/rust/feature-flags/src/flags/flag_matching_utils.rs#L556-L570 but the expected values are different. These should have the same expected values.